pulseapi/urls.py

```diff
@@ -20,9 +20,6 @@
 from ajax_select import urls as ajax_select_urls
 
 from pulseapi.profiles.views import UserProfileAPIView
-from pulseapi.utility.newsletters.signup import (
```

Comment:
change history of this file: https://github.com/mozilla/network-pulse-api/commits/master/pulseapi/urls.py